pkg/resources/resource_sink_kafka.go

```diff
@@ -24,7 +24,7 @@ var sinkKafkaSchema = map[string]*schema.Schema{
 		Elem:        "from",
 		Description: "The name of the source, table or materialized view you want to send to the sink.",
 		Required:    true,
-		ForceNew:    true,
+		ForceNew:    false,
```

Comment:
By setting the `ForceNew` flag to false, we will no longer have to drop and re-create the sink if the `from` attribute changes.